data/geonode_regions_to_administrative_divisions_code.csv

```diff
@@ -0,0 +1,4 @@
+"Turkey",233,249
+"Indonesia",115,116
+"Global",1,116
+"Global",1,249
```

Comment:
Agreed, but I'd say it depends on how @ingenieroariel is gonna generate the csv file.
Any hint, Ariel ?